### PR TITLE
Update domain lisk.io to lisk.com - Closes #3815

### LINKED ISSUE #3815: Update domain lisk.io to lisk.com
### Description

Update domain lisk.io to lisk.com

### Acceptance Criteria

- All the reference should be updated to lisk.com https://github.com/LiskHQ/lisk-desktop/search?q=lisk.io

### PR DESCRIPTION
**Pull request by ManuGowda**
### What was the problem?

This PR resolves #3815

### How was it solved?

- Updated domain reference from `io` to `com`
- Updated Lisk explorer reference to community explorer

### How was it tested?

- Navigate to all links
- Build and run the app to see if the app works

### CODE REVIEW

**Review by ikem-legend — DISMISSED** [review]

**Review by reyraa — DISMISSED** [review]

File: app/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "main": "./build/main.js",
   "author": {
     "name": "Lisk Foundation",
-    "email": "admin@lisk.io"
+    "email": "admin@lisk.com"
```
Comment (fchavant) [inline app/package.json:9]: This email adress/alias does not exist.

File: docs/DEVELOPMENT_GUIDE.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ This is a live document, please expect updates and unfinished content.
 This project and everyone participating in it is governed by the
 [Lisk Code of Conduct](https://github.com/LiskHQ/lisk/blob/development/docs/CODE_OF_CONDUCT.md). By participating, you are
 expected to uphold this code. Please report the unacceptable behavior to
-[info@lisk.io](mailto:info@lisk.io).
+[info@lisk.com](mailto:info@lisk.com).
```
Comment (fchavant) [inline docs/DEVELOPMENT_GUIDE.md:28]: As far as I can tell neither info@lisk.io nor info@lisk.com exist.

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "Lisk",
   "version": "2.0.2",
-  "author": "Lisk Foundation <admin@lisk.io>, lightcurve GmbH <admin@lightcurve.io>",
+  "author": "Lisk Foundation <admin@lisk.com>, lightcurve GmbH <admin@lightcurve.io>",
```
Comment (fchavant) [inline package.json:4]: Same as above.

File: src/components/shared/errorBoundary/index.js [hunk 4]
```diff
@@ -24,7 +24,7 @@ class ErrorBoundary extends React.Component {
   render() {
     const { t } = this.props;
     const getMailReference = () => {
-      const recipient = 'hubdev@lisk.io';
+      const recipient = 'hubdev@lisk.com';
```
Comment (fchavant) [inline src/components/shared/errorBoundary/index.js:27]: Neither hubdev@lisk.io nor hubdev@lisk.com exist.

File: src/components/screens/login/networkSelector/networkSelector.js [hunk 5]
```diff
@@ -142,7 +142,7 @@ const NetworkSelector = ({
                       onChange={onChangeInput}
                       name="customNetwork"
                       value={address}
-                      placeholder="e.g. https://mainnet-service.lisk.io or 192.168.0.1:4000"
+                      placeholder="e.g. https://mainnet-service.lisk.com or 192.168.0.1:4000"
```
Comment (fchavant) [inline src/components/screens/login/networkSelector/networkSelector.js:145]: Why use mainnet-service.lisk.com here but service.lisk.com above and below? That seems inconsistent.

File: src/constants/piwik.js [hunk 6]
```diff
@@ -1,5 +1,5 @@
 export default {
-  URL: 'https://matomo.lisk.io',
+  URL: 'https://matomo.lisk.com',
```
Comment (fchavant) [inline src/constants/piwik.js:2]: matomo.lisk.com does not exist

File: src/utils/helpers.js [hunk 7]
```diff
@@ -60,7 +60,7 @@ export const filterObjectPropsWithValue = (object = {}, value) => (
 export const getErrorReportMailto = ({
   error = 'Unknown error occurred', errorMessage, networkIdentifier, serviceUrl, liskCoreVersion,
 }) => {
-  const recipient = 'hubdev@lisk.io';
+  const recipient = 'hubdev@lisk.com';
```
Comment (fchavant) [inline src/utils/helpers.js:63]: Same as above.

**Review by fchavant — CHANGES_REQUESTED** [review]

**Review by reyraa — APPROVED** [review]

**Review by ikem-legend — APPROVED** [review]

**Pull request merged**